sensu/plugins/metric-os-api.py

```diff
@@ -0,0 +1,178 @@
+#
+# return CRITICAL if any commands fail
+# return WARNING if no commands fail, but one or more timeout
+#
+#
+# Jose L Coello Enriquez <jlcoello@us.ibm.com>
+
+from argparse import ArgumentParser
+import re
+import sys
+import time
+import subprocess
+import socket
+import os
+import json
+import requests
+
+STATE_OK = 0
+STATE_WARNING = 1
+STATE_CRITICAL = 2
+
+state = STATE_OK
+
+metric_data = ''
+
+DEFAULT_SCHEME = '{}'.format(socket.gethostname())
+
+
+def run(command):
+    p = subprocess.Popen(command, stdout=subprocess.PIPE,
```

Comment:
Deleted function run, was not being used in the rest of the code, I must have left it in there and forgot to delete it later